packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -145,11 +145,11 @@ export class LayeredRuntimeResolverCreator {
       resolvers.Subscription[operation] = {
         //tslint:disable-next-line: no-any
         subscribe: (_: any, __: any, context: GraphbackContext) => {
-          if (!context.services || !context.services[modelName]) {
+          if (!context.graphback || !context.graphback[modelName]) {
             throw new Error(`Missing service for ${modelName}`);
           }
 
-          return context.services[modelName].subscribeToUpdate({}, context);
+          return context.graphback[modelName].subscribeToUpdate({}, context);
```

Comment:
exposing as graphback to avoid conflicts + make it easy to find all related items.